cdap-ui/app/cdap/services/ThemeHelper.ts

```diff
@@ -17,7 +17,7 @@
 import cssVars from 'css-vars-ponyfill';
 import { objectQuery, isNilOrEmpty } from 'services/helpers';
 import isColor from 'is-color';
-import isBoolean from 'lodash/isBoolean';
+import { isBoolean } from 'lodash';
```

Comment:
Is this required? This will not be able to extract the `isBoolean` module alone. This will include the whole lodash module.

Reply:
will revert this back. This was from the tslint error I mentioned